travel-sponsorship.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes Contributor Travel Fund
+
+## Purpose
+
+To establish and administer a fund for members of the Kubernetes upstream contributor community to travel and particiate in KubeCon/CloudNativeCon.
+
+## Requirements
+
+ - Candidates must be a [member of the Kubernetes GitHub organization][membership] or working towards it. Exceptions: TODO
+ - Those requesting funds must indicate that they do not have funding available from another source, such as an employer or the event itself that might cover costs for presenters.
```

Comment:
@castrojo @parispittman Can they request partial funding? (just one aspect of the travel or lodging)